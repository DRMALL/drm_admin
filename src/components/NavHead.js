

import React from 'react'
import handleLogout from '../actions/handleLogout'

const NavHead = () => (
  <div className='nav-head-container' >
      <div className='nav-head-left nav-head-block' >admin</div>
      <div className='nav-hed-right nav-head-block' onClick={ handleLogout }  >退出</div>
  </div>
  )

export default NavHead
