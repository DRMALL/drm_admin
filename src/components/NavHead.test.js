

import NavHead from './NavHead'
import renderer from 'react-test-renderer'
import React from 'react'

it('中部导航渲染test', () =>{
  const tree = renderer.create(<NavHead/>)
  expect(tree).toMatchSnapshot()
})
