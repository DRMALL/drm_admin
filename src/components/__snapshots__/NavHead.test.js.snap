// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`中部导航渲染test 1`] = `
<div
  className="nav-head-container"
>
  <div
    className="nav-head-left nav-head-block"
  >
    admin
  </div>
  <div
    className="nav-hed-right nav-head-block"
    onClick={[Function]}
  >
    退出
  </div>
</div>
`;
